### ISSUE TITLE
user appears to be able to set their own "cost/value"?

### ISSUE THREAD

**Issue by danmarsden** [Snickser/moodle-paygw_yookassa#74]
https://github.com/Snickser/moodle-paygw_yookassa/blob/main/pay.php#L44

https://github.com/Snickser/moodle-paygw_yookassa/blob/main/pay.php#L55

I haven't fully traced this, but it looks like if the course cost is set to be $50 the user can decide to just pay $1 by passing "1" in to the $costself field, which will raise an invoice for that value - take them to Yookasa for payment, then on sucessful payment it will give them access?

I could be wrong though - I didn't fully trace the full process.

**Comment by Snickser** [comment]
Yes, this is specified in the settings and checked, payments can be of two types a) fixed b) any amounts from minimum to maximum (like donations), limits are set in the settings.

**Comment by danmarsden** [comment]
cool - if you could also drop some info into the readme and plugins db description that state that it contains a feature to allow users to choose how much they pay for the item along with the fix you've just made that should be enough!
